libs/service-portal/settings/access-control/src/screens/Access/Access.tsx

```diff
@@ -209,153 +212,158 @@ function Access() {
+                variant="text"
                 loading={updateLoading}
-                type="submit"
+                onClick={() => setSaveModalOpen(true)}
                 icon="checkmark"
               >
                 {formatMessage({
-                  id: 'service.portal.settings.accessControl:access-save',
+                  id: 'sp.settings-access-control:access-save',
                   defaultMessage: 'Vista aðgang',
                 })}
               </Button>
+              {saveModalOpen && (
+                <AccessModal
+                  id="access-save-modal"
+                  title={formatMessage({
+                    id: 'sp.settings-access-control:access-save-modal-content',
+                    defaultMessage: 'Ertu viss um að þú viljir veita umboðið?',
+                  })}
+                  //
+                  text={
+                    `${authDelegation?.to?.name}  mun fá umboð fyrir eftirfarandi:` ||
+                    ''
+                  }
+                  scopes={scopes as ScopeTag[]}
+                  onClose={() => setSaveModalOpen(false)}
+                  onCloseButtonText={formatMessage({
+                    id: 'sp.settings-access-control:access-save-modal-cancel',
+                    defaultMessage: 'Hætta við',
+                  })}
+                  onSubmitColor="blue"
+                  onSubmit={() => onSubmit()}
+                  onSubmitButtonText={formatMessage({
+                    id: 'sp.settings-access-control:access-save-modal-confirm',
+                    defaultMessage: 'Já, ég vil vista umboðið',
+                  })}
+                />
+              )}
             </Inline>
           </Box>
 
           <div className={styles.datePickerFix}>
-            <T.Table>
-              <T.Head>
-                <T.Row>
-                  <T.HeadData>
-                    <Text variant="small" color="blue600">
+            <Box>
+              <Hidden below="md">
+                <GridRow className={styles.row}>
+                  <GridColumn
+                    span={['12/12', '12/12', '3/12']}
+                    paddingBottom={2}
+                    paddingTop={2}
+                    className={styles.column}
+                  >
+                    <Text variant="medium" fontWeight="semiBold">
                       {formatMessage({
-                        id:
-                          'service.portal.settings.accessControl:access-access',
+                        id: 'sp.settings-access-control:access-access',
                         defaultMessage: 'Aðgangur',
                       })}
                     </Text>
-                  </T.HeadData>
-                  <T.HeadData>
-                    <Text variant="small" color="blue600">
+                  </GridColumn>
+                  <GridColumn
+                    span={['12/12', '12/12', '4/12', '5/12']}
+                    className={styles.column}
+                  >
```

Comment:
Can we make the column titles better align horizontally with each cell?

Reply:
wow - yes ✅ fixed